src/composition.jl

```diff
@@ -87,7 +87,7 @@ function A_mul_B!(y::AbstractVector, A::CompositeMap, x::AbstractVector)
         A_mul_B!(dest, A.maps[1], x)
         source = dest
         if N>2
-            dest = Array{T}(undef, size(A.maps[2],1))
```

Comment:
There are probably many lines with missing spaces, e.g. line 93 below as well. Not a comment, just a thought.